x/stakeibc/keeper/msg_server_register_host_zone.go

```diff
@@ -60,6 +63,21 @@ func (k Keeper) RegisterHostZone(goCtx context.Context, msg *types.MsgRegisterHo
 		return nil, err
 	}
 
+	// add this host zone to unbonding hostZones, otherwise users won't be able to unbond
+	// for this host zone until the following day
+	epochUnbondingRecord, found := k.recordsKeeper.GetLatestEpochUnbondingRecord(ctx)
```

Comment:
will this error out after genesis if there's no EpochUnbondingRecords? Or do we have a guarantee that an unbonding record will wind up being created before this could ever get called?